agent/src/podman/cli_command.rs

```diff
@@ -0,0 +1,231 @@
+use std::process::Stdio;
+use tokio::io::AsyncWriteExt;
+use tokio::process::Command;
+
+#[cfg(test)]
+pub use tests::MockCliCommand;
+
+pub struct CliCommand<'a> {
```

Comment:
I do not see that CliCommand is something podman specific. Couldn't we move it into common for reuse-ability (if someone needs to execute cli commands for another runtime impl)?

Reply:
Fixed.